Automattic/vip-go-mu-plugins
# Hide Akismet admin notices

## Description

This PR removes some admin notices that Akismet would display when it was not activated. Given that we want to move towards activating Akismet automatically, we don't want to show the user buttons that are not actionable.

We are removing notices such as:

<img width="1731" alt="Screen Shot 2021-06-14 at 11 51 23 AM" src="https://user-images.githubusercontent.com/7188409/121873630-db704d00-cd06-11eb-89d6-a97a45a8672a.png">

The PR includes a small refactor to remove Akismet pages as well.

## Changelog Description

### Plugin Updated: Akismet

Removed Akismet notices.

## Checklist

Please make sure the items below have been covered before requesting a review:

- [ x] I've created a changelog description that aligns with the provided examples.

## Steps to Test

Example:

1. Check out PR.
2. Go to `wp-admin` > `Plugins`
3. If Akismet is not active but Jetpack is, you shouldn't see the admin notice.
